hack/make-rules/test-cmd.sh

```diff
@@ -101,7 +99,7 @@ function run_kube_controller_manager() {
 # Exports:
 #   SUPPORTED_RESOURCES(Array of all resources supported by the apiserver).
 function create_node() {
-  kubectl create -f - -s "http://127.0.0.1:${API_PORT}" << __EOF__
+  kubectl create -f - -s "http://127.0.0.1:${SECURE_API_PORT}" << __EOF__
```

Comment:
http + SECURE_API_PORT won't work (applies everywhere)

Reply:
yeah, i want to sweep everything :-)